test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataProviderTest.cs

```diff
@@ -869,6 +948,21 @@ private class RequiredMember
             public string RequiredProperty { get; set; }
         }
 
+        [BindNever]
+        private class BindingBehaviorModel
+        {
+            [BindRequired]
+            public int BindRequiredOverride { get; set; }
+
+            [BindingBehavior(BindingBehavior.Optional)]
+            public int BindingBehaviorOptionalOverride { get; set; }
+
+            [BindingBehavior(BindingBehavior.Never)]
+            public int BindingBehaviorNever { get; set; }
+
+            public int BindingBehaviorNeverSetOnClass { get; set; }
```

Comment:
I'm avoiding using the term inherited for anything other than class -> class inheritance.